spec/features/autocomplete_typeahead_spec.rb

```diff
@@ -27,6 +27,30 @@
         expect(featured_image.iiif_image_id).to eq 'https://purl.stanford.edu/gk446cj2442/iiif/annotation/gk446cj2442_1'
         expect(featured_image.iiif_tilesource).to eq 'https://stacks.stanford.edu/image/iiif/gk446cj2442%2Fgk446cj2442_05_0001/info.json'
       end
+
+      it 'instantiates the multi-image selector when an multi-image item is chosen in the typeahead (and again on edit)' do
+        # Poltergeist doesn't play nice with croppers behind tabs. This is not the case in real browsers
+        allow(Spotlight::Engine.config).to receive(:exhibit_themes).and_return(['default'])
```

Comment:
This feels unsatisfactory, but the only other way would be to skip the test entirely.  Otherwise a javascript error is thrown (from somewhere down in the leaflet-iiif code) that is not reproducible in a real browser.